frontend/src/DevelopersPortfolio/LandingPage/LandingPage.tsx

```diff
@@ -28,6 +30,7 @@ export const LandingPage = () => {
     }, 10);
     return () => clearInterval(timer);
   }, []);
+  // Put in a hook
```

Comment:
nit: we can add an issue number here in the comment so we can keep track of it 

eg. `"//TODO: #999 - encapsulate this in a hook`